managers/coverFetchers/publisher_backlist_cover_fetcher.py

```diff
@@ -0,0 +1,80 @@
+import hashlib
+import io
+import os
+
+import PIL
+import pypdf
+import requests
+import requests.exceptions
+
+from logger import create_log
+from managers.coverFetchers.abstractFetcher import AbstractFetcher
+
+logger = create_log(__name__)
+
+
+class PublisherBacklistCoverFetcher(AbstractFetcher):
+
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.session = requests.Session()
+        self.bucket = os.environ["FILE_BUCKET"]
+        self.pdf_url: str | None = None
+
+
+    def hasCover(self):
+        for value, source in self.identifiers:
+            if source != "hathi":
+                continue
+
+            url = (
+                f"https://{self.bucket}.s3.us-east-1.amazonaws.com"
+                f"/titles/publisher_backlist/Schomburg/{value}/{value}.pdf"
+            )
+
+            response = self.session.head(url)
+
+            try:
+                response.raise_for_status()
+            except requests.exceptions.HTTPError:
+                continue
+            except Exception as e:
+                logger.exception("Unexpected error when fetching PDF")
+                continue
+            else:
+                self.pdf_url = url
+                return True
+
+        return False
+
+    def downloadCoverFile(self) -> bytes:
+        with io.BytesIO() as stream:
+            response = self.session.get(self.pdf_url, stream=True)
+            response.raise_for_status()
```

Comment:
Do you think we should include a try/catch here?

Reply:
download_cover_file will also change with the PR I have up